modules/engines/engine-hdb/src/main/java/com/sap/xsk/hdb/ds/module/XSKHDBModule.java

```diff
@@ -101,13 +103,19 @@ private static void bindParsersToFileExtension(Map<String, XSKDataStructureParse
         new XSKHDBTableFunctionLogger()
     );
 
+    XSKHDBProcedureParser hdbProcedureParser = new XSKHDBProcedureParser(
+            new XSKDataStructuresSynchronizer(),
```

Comment:
It would be better to reuse the instances passed for the `XSKHDBTableFunctionParser` and `XSKHDBProcedureParser` constructors. Please, move them to separate variables